app/src/main/java/cz/ackee/sample/di/DIContainer.kt

```diff
@@ -0,0 +1,54 @@
+package cz.ackee.sample.di
+
+import cz.ackee.ackroutine.CoroutineOAuthManager
+import cz.ackee.retrofitadapter.AckroutineCallAdapterFactory
+import cz.ackee.sample.App
+import cz.ackee.sample.interactor.ApiInteractorImpl
+import cz.ackee.sample.interactor.IApiInteractor
+import cz.ackee.sample.model.Logouter
+import cz.ackee.sample.model.rest.ApiDescription
+import cz.ackee.sample.model.rest.AuthApiDescription
+import io.appflate.restmock.RESTMockServer
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+
+/**
+ * Simple DI container that provides dependencies
+ */
+class DIContainer(val app: App) {
+
+//    val rxOAuthManager = RxOAuthManager(
```

Comment:
can be removed?

Reply:
next PR focuses on coroutine integration of all the commented out stuff, this PR is just a hard copy of RxOAuth example without rxjava/rxoauth dependency